learning-development/accessibility.qmd

```diff
@@ -52,6 +51,48 @@ For more advice about colour in charts and visualisations specifically, includin
+
+### a. Consult the Spreadsheet Accessibility Guidance
+
+Before diving into the spreadsheet creation process, it's essential to familiarize yourself with the official [spreadsheet accessibility](https://analysisfunction.civilservice.gov.uk/policy-store/releasing-statistics-in-spreadsheets/) guidance available on the Analysis Function Guidance Hub. This resource offers comprehensive insights into best practices for ensuring that your spreadsheets are accessible to all users.
+
+### Accessibility and consequences
+
+Public sector spreadsheets must meet the AA level of the Web Content Accessibility Guidelines (WCAG) 2.2, as required by law under The Public Sector Bodies (Websites and Mobile Applications) Accessibility Regulations 2018. This guidance distinguishes between mandatory actions needed to comply with legal accessibility standards and best practices that enhance usability. Non-compliance can lead to legal complaints, making it crucial for content creators to understand and mitigate these risks.
+
+In addition to accessibility, the guidance also addresses usability and machine readability. While these areas often overlap with accessibility, there are instances where they may conflict. Depending on user needs, it might be necessary to produce separate versions of a spreadsheet—one for human use and another optimized for machine readability—to ensure both accessibility and functionality are maintained.
+
+The [guidance](https://analysisfunction.civilservice.gov.uk/policy-store/releasing-statistics-in-spreadsheets/) is extensive, including sections how to format sheets and cells, accessible titles, using formula & macros and much more!
+
+## b. Automate Accessibility During the Coding Process
+
+Automation is a powerful way to ensure that accessibility is consistently applied throughout the spreadsheet creation process, especially when working with large datasets or producing numerous spreadsheets. By integrating accessibility into the coding process, you can streamline your workflow and reduce the chances of missing critical accessibility features.
+
+Here’s how you can automate accessibility using specific tools in Python and R:
+
+1. For Python Users – gptables Package:
+
+- Overview: The ``gptables`` package is designed to help Python users create accessible tables within spreadsheets. This package automates the formatting of tables to meet accessibility standards, ensuring that key elements such as table headers, data descriptions, and structures are correctly implemented.
+- Benefits: By using ``gptables``, you can automatically generate tables that are readable by screen readers, properly formatted for navigation, and compliant with accessibility regulations. This reduces the manual effort required to adjust table formats and enhances consistency across multiple spreadsheets.
+
+2. For R Users – ``a11ytables`` Package:
```

Comment:
I think move the R section before the python section. And also change this line to a heading:

#### Producing accessible tables in R using `a11ytables`